src/modules/project/pages/projectView/views/projectMainBody/components/RewardCard.tsx

```diff
@@ -64,19 +70,48 @@ export const RewardCard = ({ reward, count, ...rest }: Props) => {
           {reward.description}
         </Text>
       </VStack>
-      <Button
-        variant="primaryNeutral"
-        size="sm"
-        height={'40px'}
-        onClick={(e) => {
-          rest.onRewardClick?.(e)
-        }}
-        isDisabled={!isRewardAvailable}
-      >
-        <Text fontSize={16} fontWeight={500} isTruncated>
-          {t('Select item')}
-        </Text>
-      </Button>
+      <HStack>
+        {!isLaunch ? (
+          <Button
+            variant="primaryNeutral"
+            size="sm"
+            height={'40px'}
+            flexGrow={1}
+            onClick={(e) => {
+              rest.onRewardClick?.(e)
```

Comment:
maybe remove the rest, we use rest only for direct passage of props to other components. This is not wrong, but just less consistent.